packages/block-editor/src/components/rich-text/index.js

```diff
@@ -485,9 +485,7 @@ PrivateRichText.isEmpty = ( value ) => {
  * @see https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/rich-text/README.md
  */
 const PublicForwardedRichTextContainer = forwardRef( ( props, ref ) => {
-	return (
-		<PrivateRichText ref={ ref } { ...props } disableEditing={ false } />
-	);
+	return <PrivateRichText ref={ ref } { ...props } readonly={ false } />;
```

Comment:
When do you think it could get public?

Reply:
@ellatrix what are your thoughts on this? It seems like a reasonably low-risk thing to add to the public API. Should we leave it private for a plugin release or two and aim to have it public for 6.6? 

@retrofox I assume it is not much use to you until it is in a WP core release as many woo users won't be using the GB plugin, so no panic to have it public before 6.6?

Reply:
@glendaviesnz @retrofox Are you stil interested in making the `readOnly` attribute public? Does Woo have a use case for it?

It's been 8 months since it's been added, 6.6 and 6.7 releases passed by and now we're working on 6.8.